server/helpers/db.js

```diff
@@ -1,7 +1,21 @@
+const { Pool } = require("pg");
+const config = require("../__db/db-config")[process.env.NODE_ENV || "development"];
+
+const pool = new Pool(config);
+
+const getHotels = () => {
```

Comment:
I think a db helper should just expose methods to transact with the DB. `getHotels` feels more like it belongs in a a service specifically for interacting with hotel entities.